wiki/People/Community_Contributors/en.md

```diff
@@ -9,102 +8,128 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni) which are previous
 
 ## 2013
 
-### August
-
-*Also see the forum thread: [osu! community contributors](/community/forums/posts/2489459).*
+<!-- link is dead https://osu.ppy.sh/community/forums/posts/2489459 -->
+*Also see the forum thread: [osu! community contributors](https://web.archive.org/web/20131003162259/https://osu.ppy.sh/forum/p/2489459 "Wayback Machine").*
```

Comment:
```suggestion
*For the forum thread, see: [osu! community contributors](https://web.archive.org/web/20131003162259/https://osu.ppy.sh/forum/p/2489459 "Wayback Machine")*
```

Reply:
I was about to say that i can recover this post from the wasteland, probably better than linking to the web archive.